src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -182,10 +195,10 @@
 
 (defn add-on-press-handler-to-categorized-chats
   [community-id categorized-chats]
-  (reduce-kv (fn [acc category chats]
-               (assoc acc category (add-on-press-handler-to-chats community-id chats)))
-             {}
-             categorized-chats))
+  (let [add-on-press (partial add-on-press-handler-to-chats community-id)]
+    (map (fn [[category v]]
```

Comment:
what is `v`? 🤔

Reply:
value :) probably should rename it I guess :)

Reply:
I know it's a questionable practice, but it's very idiomatic in Clojure to use `v` to mean `value`, especially in short functions or when iterating over maps like the code above. Also when writing more general functions that don't care about the exact type of the value. No strong preference, maybe here it's better to use the long name. For reference, here's the list of idiomatic names https://guide.clojure.style/#idiomatic-names